v1/pipeline/stages/custom/custom-stage-spec-element-wrapper-config.yaml

```diff
@@ -1,8 +1,6 @@
 title: CustomStageSpecElementWrapperConfig
 type: object
 properties:
-  parallel:
-    $ref: parallel-step-element-config.yaml
   step:
```

Comment:
it doesnt contain step keyword also right?

Reply:
yes, removed step keyword